hydromt/components/geoms.py

```diff
@@ -47,10 +54,22 @@ def __init__(
         filename: str
             The path to use for reading and writing of component data by default.
             by default "geoms/{name}.geojson" ie one file per geodataframe in the data dictionary.
+        region_component: str, optional
+            The name of the region component to use as reference for this component's
+            region. If None, the region will be set to the union of all geometries in
+            the data dictionary.
+        region_filename: str, optional
+            The path to use for writing the region data to a file. By default
+            "geoms/geoms_region.geojson".
         """
         self._data: Optional[Dict[str, Union[GeoDataFrame, GeoSeries]]] = None
         self._filename: str = filename or self.__class__.DEFAULT_FILENAME
-        super().__init__(model=model, region_component=region_component)
+        region_filename = region_filename or self.__class__.DEFAULT_REGION_FILENAME
```

Comment:
Because `DEFAULT_REGION_FILENAME` is overriding the original `DEFAULT_REGION_FILENAME`, there should be no need for this line 67 because the base class should pick up the right one.

Reply:
Ok then we can remove indeed